TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/V20240920/Responses/GetPersonResponse.cs

```diff
@@ -1,17 +1,52 @@
+using AutoMapper.Configuration.Annotations;
 using Optional;
+using TeachingRecordSystem.Api.Infrastructure.Mapping;
 using TeachingRecordSystem.Api.V3.Core.Operations;
+using TeachingRecordSystem.Api.V3.V20240101.ApiModels;
+using TeachingRecordSystem.Api.V3.V20240606.Responses;
 using TeachingRecordSystem.Api.V3.V20240920.ApiModels;
 
 namespace TeachingRecordSystem.Api.V3.V20240920.Responses;
 
 [AutoMap(typeof(GetPersonResult))]
-[GenerateVersionedDto(typeof(V20240606.Responses.GetPersonResponse), excludeMembers: ["Alerts", "Sanctions", "Induction"])]
+[GenerateVersionedDto(typeof(V20240606.Responses.GetPersonResponse), excludeMembers: ["Alerts", "Sanctions", "Induction", "InitialTeacherTraining"])]
 public partial record GetPersonResponse
 {
     public required Option<IReadOnlyCollection<Alert>> Alerts { get; init; }
     public required Option<GetPersonResponseInduction?> Induction { get; init; }
+    public required Option<IReadOnlyCollection<GetPersonResponseInitialTeacherTraining>> InitialTeacherTraining { get; init; }
 }
 
 [AutoMap(typeof(GetPersonResultInduction))]
 [GenerateVersionedDto(typeof(V20240606.Responses.GetPersonResponseInduction), excludeMembers: ["Periods"])]
 public partial record GetPersonResponseInduction;
+
+[AutoMap(typeof(GetPersonResultInitialTeacherTraining))]
+public partial record GetPersonResponseInitialTeacherTraining
+{
+    public required GetPersonResponseInitialTeacherTrainingProvider? Provider { get; init; }
+
+    [ValueConverter(typeof(WrapWithOptionValueConverter<GetPersonResultInitialTeacherTrainingQualification, GetPersonResponseInitialTeacherTrainingQualification>))]
```

Comment:
I couldn't find a way of doing this without decorating all these properties with attributes